### PR TITLE
Fix onboarding local database creation

### PR DESCRIPTION
**Pull request by muamichali**

### CODE REVIEW

File: migrations/20171030203050_init.js [hunk 1]
```diff
@@ -91,7 +84,7 @@ const statements = [
   'ALTER TABLE person MODIFY id int(10) UNSIGNED NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=1;',
 ];
 
-exports.up = function(knex, Promise) {
+exports.up = function(knex) {
```
Comment (GalGend) [inline migrations/20171030203050_init.js]: I think promise should be required from bluebird, anyway, I have made those changes on a branch called Setup_fixes_2020 with several more changes to the setup 
 https://github.com/meirim-org/meirim/tree/Setup_fixes_2020

Comment (GalGend) [reply migrations/20171030203050_init.js]: @muamichali

Comment (muamichali) [reply migrations/20171030203050_init.js]: @GalGend Oh, I see your branch now, cool! I will close the PR

Comment (muamichali) [reply migrations/20171030203050_init.js]: @GalGend The Promise parameter is not required.. See here https://github.com/knex/knex/issues/3375#issuecomment-515701216

**Pull request closed without merging**